packages/theme/src/components/image.ts

```diff
@@ -0,0 +1,7 @@
+import type { ComponentStyle } from "@yamada-ui/core"
+
+export const Image: ComponentStyle = {
+  defaultProps: {
+    fallbackStrategy: "beforeLoadOrError",
```

Comment:
This is not theme propertiy.

Reply:
@108yen 
Am I correct in setting only ImageOptions to defaultProps?
If so, do I need to set it up?

Reply:
@cidkumagai 
Please read the article I wrote on how to use the theme.
It would be the same as `infinite-scroll-area.ts`.
https://zenn.dev/108yen/articles/d5b02cf775a0c0

Reply:
@108yen 
Thank you!
I have fixed it, please review.